.github/workflows/auto-release.yml

```
@@ -0,0 +1,117 @@
+          # Get the latest release tag, or default to v0.0.0 if no releases exist
+          LATEST_RELEASE=$(gh release list --limit 1 --json tagName --jq '.[0].tagName' 2>/dev/null || echo "v0.0.0")
+          echo "latest_release=$LATEST_RELEASE" >> $GITHUB_OUTPUT
+          echo "Latest release found: $LATEST_RELEASE"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Calculate next version
+        id: next_version
+        run: |
+          CURRENT_VERSION="${{ steps.get_latest_release.outputs.latest_release }}"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)